### DISCUSSION TITLE
Posible incompatibilidad por error detectado al instalar dependencias

### DISCUSSION THREAD

**Discussion by hrchioest** [TiendaNube/nimbus-patterns#51] [Q&A] [answered]
Buen día, se reporta este error,  estuve utilizando @nimbus-ds/styles  versión 8.0.1 y "@nimbus-ds/patterns": "1.2.1", para cambiar los menuButton del menu a Link  y las pruebas que estuve haciendo fue navegar entre los botones o links del menú y al  presionar para cerrar sesión se rompía todo. Podrían verificarlo para poder detectar este error? Les mando la captura del error que me mostraba.
![Captura de Pantalla 2023-05-04 a la(s) 13 08 26](https://user-images.githubusercontent.com/53129859/236523974-63ae0e84-7530-418e-beee-9b7f912d4fd7.png)

Al cambiar la version de @nimbus-ds/styles a la versión 8.0.0 ahi ya no se me rompía y por lo pronto se mantuvo estable.

**Comment by juniorconquista** [comment, marked as answer]
@hrchioest conforme conversamos neste caso era algumas libs dos nimbus que estavam desatualizadas no projeto.
